src/main/sensors/gyro.c

```diff
@@ -125,12 +136,16 @@ void gyroUpdate(void)
         return;
     }
 
-    if (gyroFIRTable) {
-        filterApplyFIR(gyroADC, gyroFIRState, gyroFIRTable);
-    }
-
     alignSensors(gyroADC, gyroADC, gyroAlign);
 
+    if (gyroLpfCutFreq) {
+        if (!gyroFilterStateIsSet) {
+            initGyroFilterCoefficients();
+        } else {
```

Comment:
There is no reason for skipping first iteration. The code will be 'simpler' without it.